control/control_debug_tools/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>control_debug_tools</name>
+  <version>0.1.0</version>
+  <description>The control_debug_tools package</description>
+  <maintainer email="zhe.shen@tier4.jp">Zhe Shen</maintainer>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <license>Apache License 2.0</license>
+
+  <author email="zhe.shen@tier4.jp">Zhe Shen</author>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <buildtool_depend>autoware_cmake</buildtool_depend>
+  <buildtool_depend>eigen3_cmake_module</buildtool_depend>
+
+  <build_depend>rosidl_default_generators</build_depend>
```

Comment:
Remove

Reply:
removed in c30816d8a398a220f1a6c2db0a552f3875aebca8